app/views/software_instances/_form.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= simple_form_for(software_instance) do |form| %>
+  <%= form.input :name %>
```

Comment:
This form show how simple form makes form definition simpler as it will automatically put the form fields into pattern defined in the simple form config file.